templates/ceph/index.html

```diff
@@ -305,6 +306,6 @@ <h3 class="p-heading--4">Managed Ceph</h3>
   </div>
 </section>
 <section class="p-strip--light is-deep is-bordered">
-  {% include "shared/_resources_ceph.html"%}
+  { % include "shared/_resources_ceph.html"%}
```

Comment:
Is this by Djlint? Normally there should not be a space between `{` and `%` and there should be a space between `.html"` and `%`

Reply:
Yeah, I have corrected that